Instant code Generation No Response

For some reasons, Im unable to generate a Code using the --code method. I guess the error is some kind of rate limiting.

I get two kinds of errors:
Either I get ratelimited, have to wait some time and then get this response:

`An error occurred requesting code from server; response [200]`

Or I am not forced to wait some time, but then I instantly get 

`An error occurred requesting code from server; response [429]`

No matter, which error occurs, this is always the last line in the log:

`[INFO] <impf.api.api.py:43>  {}`

So I dont seem to get any response at all.

As a User-Agent, I used the default one and `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36`

I tried it with 2 different Phonenumbers and 3 different Emails, so I dont think its because of that.
The locations I tried are 88444 Ummendorf, 88212 Ravensburg and 89073 Ulm

I also tried to wait 30 Minutes after some tries and 3 different IPs, and also from my server, nothing worked.

Also I tried it in Docker and natively on Windows, both time the same error

```
2021-05-20 18:57:06,128 - [INFO] <impf.browser.browser.py:140>  89073: Navigating to ImpfterminService
2021-05-20 18:57:08,545 - [INFO] <impf.browser.browser.py:153>  89073: Selected Bundesland: Baden-Württemberg           
2021-05-20 18:57:08,760 - [INFO] <impf.browser.browser.py:161>  89073: Selected Impfzentrum: 89073 Ulm, Zentrales Impfzentrum Ulm
2021-05-20 18:57:27,263 - [INFO] <impf.api.api.py:33>  API: Attempting to get Vermittlungscode from server              
2021-05-20 18:57:30,719 - [ERROR] <impf.api.api.py:44>  API: An error occurred requesting code from server; status code [429]                                                                                                                   
2021-05-20 18:57:30,720 - [INFO] <impf.api.api.py:45>  API: Response: {}
```

Edit: 
I tried to put the steps done in the API.py into a standalone script, and here I constistently get Statuscode 200, but no returned data

--- a/requirements.txt
+++ b/requirements.txt
@@ -10,5 +10,5 @@ pyOpenSSL==20.0.1
 requests==2.25.1
 selenium==3.141.0
 six==1.16.0
-urllib3
+urllib3==1.26.4
 zulip==0.7.1

